COMMITTING.md

```diff
@@ -45,15 +45,14 @@ In general, most commits do not have a prefix.
 
 ### Example
 
-From [whatwg/html@54585](https://github.com/whatwg/html/commit/5458513792ab00d58e6c91ba48faaa611d034a2e):
-
 ```
-Editorial: move base URL from "module script" to "script"
+Editorial: Fix formatting and typo
+
+Fixes: #35, #38, #21
```

Comment:
Should be `Fixes #35, fixes #38, fixes #21` so that github can automatically close the issues on merge.

Reply:
@zcorpan you are correct. I got stuck in meta inside of meta. Overthought it. #YoDawgMeme :-)

Hopefully this pleases you https://github.com/whatwg/meta/pull/119/commits/d740fa70ffce4964110792e276a426a973de65b3